docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -207,6 +207,27 @@ customAnnotations:
     runbook_url: https://www.google.com/?q=Watchdog
 ----
 
+== `alerts.sharedStorageClass`
+
+[horizontal]
+type:: string
+default:: ``
+
+A regular expression that matches the shared storage classes in this cluster.
+With that we mean storage classes which PVs share the same underling volume, which causes them to fill up at the same rate.
```

Comment:
We tend not to use phrasing that you'd find in a research paper in our reference documentation.

```suggestion
A shared storage class is a storage class for which PVs share the same underlying volume, which causes them to fill up at the same rate.
```